modin/pandas/test/utils.py

```diff
@@ -110,6 +111,9 @@
     },
 }
 
+test_data["with_index_column"] = copy.copy(test_data["int_data"])
```

Comment:
Can we add a comment above this about what it is testing

Reply:
Sure, I added a comment and a TODO to replace this kind of initialization with normal one when #1481 is resolved.